src/main/java/org/opensearch/performanceanalyzer/rca/net/tasks/FlowUnitRxTask.java

```diff
@@ -76,9 +74,7 @@ public void run() {
             LOG.warn(
                     "Dropped a flow unit because the vertex buffer was full for vertex: {}",
                     vertex);
-            StatsCollector.instance().logMetric(RcaConsts.VERTEX_BUFFER_FULL_METRIC);
```

Comment:
Do we want emit equivalent metric here with the other framework?

Reply:
So, we have a repetitive metric here. We already emit `RCA_VERTEX_RX_BUFFER_FULL_ERROR` inside [receivedFlowUnitStore.enqueue()](https://github.com/opensearch-project/performance-analyzer-rca/blob/main/src/main/java/org/opensearch/performanceanalyzer/rca/net/ReceivedFlowUnitStore.java#L84)